collection-scripts/gather_priority_and_fairness

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="/must-gather"
+NAMESPACE_PATH="namespaces/openshift-kube-apiserver"
+POD_PATH="pods/kube-apiserver/kube-apiserver"
+APF_PATH="${BASE_COLLECTION_PATH}/${NAMESPACE_PATH}/${POD_PATH}/api_priority_and_fairness"
+
+# If the cluster version is < 4.5 skip this test
+VERSION=$(oc get clusterversion version -o jsonpath='{.status.desired.version}')
+COMPARE=$(echo -e "4.5\n${VERSION}" | sort -V | head -n1)
+if [[ ${COMPARE} != "4.5" ]]; then
```

Comment:
this PR is targeting `master` branch, not `4.5` branch. I think we don't have to take the cluster version into account

Reply:
I'd leave it, you can always use newer image with on an older cluster, so it won't hurt.

Reply:
Yes it does return just list of available endpoints in case its unavailable < 4.5. I did it as we discussed it but do we need to keep version check? Honestly it just adds unnecessary complexity and further requirements. What do you think? Shall I remove that part? It wont break must-gather not having it. @tkashem @p0lyn0mial @soltysh

Reply:
> do we need to keep version check

yes, based on the input from @soltysh we know that users might run arbitrary images (must-gather) against clusters.

Reply:
decided to keep it since older versions of cluster can still run newer must-gather